docs/contributing/translations.md

```diff
@@ -59,6 +59,10 @@ To edit an already approved translation string follow the next steps:
 
 ![Weblate Browse translations](../assets/img/translations/weblate-browse-translations.png)
 
-Saved editions will get the status "Waiting for review". Suggestions will get the status "Approved strings with suggestions". Our team will periodically check strings waiting for review and, if considered correct, will approve them.
+When collaborating with translations, Weblate will automatically create a Pull Request in the project repository with all translations added.
```

Comment:
Maybe we can comment that if you have the github account linked to Weblate your github account will appear in the PR? I don;t know if this add some value to know.